extensions/merge-conflict/package.json

```diff
@@ -83,6 +83,20 @@
         "command": "merge-conflict.compare"
       }
     ],
+    "menus": {
+      "scm/resourceState/context": [
+        {
+          "command": "merge-conflict.accept.all-current",
+          "when": "scmProvider == git && scmResourceGroup == merge",
```

Comment:
This is not specific to git. @joaomoreno Should we do this for any files in 'merge'?